service/telemetry.go

```diff
@@ -160,14 +159,12 @@ func (tel *colTelemetry) initOpenCensus(level configtelemetry.Level, instanceID
 
 	// Until we can use a generic metrics exporter, default to Prometheus.
 	opts := prometheus.Options{
-		Namespace: getMetricsPrefix(),
+		Namespace: col.service.buildInfo.Command,
```

Comment:
I think we should remove this completely. Probably better for the moment to hardcode as "otelcol" and in a future PR we can look into removing it. I am not sure we should give the option to configure this as Command.

Reply:
Restored to "otelcol".  I think it beneficial to provide more information about the running collector but I'd rather get the foundation of this capability landed than have it fully formed.